src/tests/conference.test.ts

```diff
@@ -0,0 +1,56 @@
+import {Conference} from '@app/conference'
+import {
+	dummyOrganizer,
+	dummyOrganizer2,
+	dummyAuthor1,
+	session,
+	session2,
+	article,
+	article2
+} from '@tests/dummies'
+
+describe('Test conferences use cases', () => {
+	test('Create a new conference and add Chair and Session correctly', () => {
```

Comment:
Cuidado con los nombres de los tests. Es dificil de interpretar el correctamente. La descripcion del test debería ser intuitivo al testear sin caer a ver el codigo porque si el codigo esta mal entonces no sabemos si el test esta bien.

Fijate de con el mismo nombre expresar lo que queres probar. Si te sale mejor con codigo primero, fijate los expect que dejaste. Ej. _Conference sessions should match those added at first._ o _ Get authors should match those added._